libsrc/osic1p/cputc.s

```diff
@@ -36,10 +41,13 @@ L1:     cmp     #$0D            ; LF?
 cputdirect:
```

Comment:
If I wanted to have you remove the scrolling support from your CONIO implementation I would have already stated so before merging your recent pull request ;-)

I just wanted to provide you with an overall overview how things are usually setup on other (comparable) targets. This pull request seems to invest quite some effort in scrolling for different screen sizes so I wanted to make sure you know that scrolling isn't required in the first place - and explain why this was decided this way.

Regarding the single user: Is he using CONIO features like cursor positioning and/or text attributes? If not then he might actually rather be a POSIX I/O user only driven to your CONIO because of POSIX I/O missing ;-)

Regarding POSIX I/O based on CONIO: Other targets with well known firmware entry points for character output tend to have a redirection mechanism of one kind or another allowing BASIC programs to print another terminal, a printer, a different video adapter, <...>. The idea of mapping POSIX I/O to those firmware entry points for character output is to allow those redirection mechanisms to work like with BASIC programs.